test/api_integration/apis/home/sample_data.ts

```diff
@@ -64,7 +67,7 @@ export default function ({ getService }: FtrProviderContext) {
             .expect(200);
 
           expect(resp.body).to.eql({
-            elasticsearchIndicesCreated: { kibana_sample_data_flights: 13059 },
+            elasticsearchIndicesCreated: { kibana_sample_data_flights: 13014 },
```

Comment:
Reflect modification to sample data in number of documents that would get indexed.